mtp_cashbook/templates/cashbook/transaction_batch_list.html

```diff
@@ -12,62 +12,83 @@
     data-unload-msg="{% trans "You are leaving this page, but have ticked payments. &#10;&#10;You must click &#8216;I&#8217;ve credited payments in NOMIS&#8217; if payments have been credited." %}">
     {% csrf_token %}
 
-    <div class="totals js-StickyHeader"  id="tour-step-total">
-      <div class="container js-RunningTotal" data-label="{% trans 'Total payments you have credited: ' %}">
-        <p>
-          {% blocktrans with total=total|currency highlight="<strong>&pound;" highlightend="</strong>" %}
-            Control total: {{ highlight }}{{ total }}{{ highlightend }}
-          {% endblocktrans %}
-        </p>
-      </div>
-    </div>
 
     {% if object_list %}
-    {{ form.errors }}
-
-    {% include "cashbook/includes/batch_buttons.html" %}
+      {% if form.errors or form.non_field_errors %}
```

Comment:
This block could be extracted out into an include and added to [money-to-prisoners-common](http://github.com/ministryofjustice/money-to-prisoners-common) if it can be used across all 3 apps.

A slight variation is already being used in the [login template](https://github.com/ministryofjustice/money-to-prisoners-common/blob/4343e219e86acfa55ef0ac2ea630e767f94a8d48/templates/auth/login.html#L14-L35).